### DISCUSSION TITLE
@swc/jest vs @swc-node/jest

### DISCUSSION THREAD

**Discussion by stefee** [swc-project/swc#2238] [Q&A] [answered]
Is there a reason to use one over the other? Are they two implementations of the same idea, or different use case?

- https://www.npmjs.com/package/@swc-node/jest
- https://www.npmjs.com/package/@swc/jest

P.S. It seems to me that the only difference is in the configuration schema, and `@swc-node/jest` has more useful (and somewhat opinionated) default configuration than `@swc/jest`. Is there a need to maintain both of these projects going forward?

**Comment by Brooooooklyn** [comment, marked as answer]
@swc-node/* is a simple abstraction on @swc/* which support `tsconfig.json` style configuration, there is no other differences.

**Reply by stefee** [reply]
Thanks

**Reply by bfaulk96** [reply]
@Brooooooklyn will `@swc-node` be regularly updated to support new features and updates as `@swc` releases them? Just curious, as the mentioned abstraction to support `tsconfig.json` seems really nice

**Reply by Brooooooklyn** [reply]
Yes, I'll track the update from @swc/* in @swc-node/*
